src/Stream/StreamerInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
```

Comment:
Maybe it makes sense to have an abstract class which knows which endpoint gets called and decide which streamer to use. We probably need an streamer for each call.